packages/launchpad/src/settings/project/ExperimentRow.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import ExperimentRow from './ExperimentRow.vue'
+import { experiments } from './projectSettings'
+
+describe('<ExperimentRow />', () => {
+  it('renders an experiment with a status', { viewportWidth: 800, viewportHeight: 600 }, () => {
+    cy.mount(() => (
+      <ExperimentRow class="resize overflow-auto w-600px p-12" experiment={ {
+        ...experiments[0],
+        enabled: true
+      }} />
+    ))
+
+    cy.findByText('Enabled').should('be.visible')
```

Comment:
If there's a bug in Cypress testling library, should we just use the usual get/contains combo? Alternatively we could write out own `findByText` (which likely would not have the bug, just combo of `get` and `contains`)

Reply:
I don't think there's a bug in the Cypress testing library.